flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/testutils/junit/extensions/retry/strategy/RetryStrategy.java

```diff
@@ -27,11 +27,17 @@ public interface RetryStrategy {
     void stopFollowingAttempts();
 
     /**
-     * Handle the exception after the test execution.
+     * Handle an exception that occurred during the annotated test attempt.
+     *
+     * <p>This method can swallow the exception to pass the test.
      *
      * @param testName the test name
      * @param attemptIndex test attempt index that starts from 1
      * @param throwable the throwable that the test case throws
+     * @throws org.opentest4j.TestAbortedException When handling a test attempt failure, throwing
```

Comment:
may be it makes sense to use link here e.g. `{@link TestAbortedException}`

Reply:
that's actually a good point. :+1:

Reply:
:D Luckily the `@throws` annotation automatically links to the exception when it makes sense.   In this case, TestAbortedException is outside the generated Flink javadocs in any case, I'm not sure external links to Jupiter have been configured.

Reply:
Ah true, I haven't considered that it's outside of the Flink space. :+1: